dev/doc/debugging.md

```diff
@@ -22,65 +22,10 @@ Debugging from Coq toplevel using Caml trace mechanism
   printers too.
 
 
-Debugging with ocamldebug from Emacs
-====================================
+Debugging with ocamldebug from Emacs or command line
+====================================================
 
-   Requires [Tuareg mode](https://github.com/ocaml/tuareg) in Emacs.\
-   Coq must be configured with `-local` (`./configure -local`) and the
-   byte-code version of `coqtop` must have been generated with `make byte`.
-
-   1. M-x camldebug
-   2. give the binary name bin/coqtop.byte
-   3. give ../dev/ocamldebug-coq
-   4. source db  (to get pretty-printers)
-   5. add breakpoints with C-x C-a C-b from the buffer displaying the ocaml
-      source
-   6. get more help from ocamldebug manual
-         run
-	 step
-         back
-         start
-	 next
-	 last
-	 print x (abbreviated into p x)
-	 ...
-   7. some hints: 
-
```

Comment:
@SkySkimmer , these hints, even if likely outdated, are what I mean; IMHO they are nice, as they point to specific places in Coq's codebase.

Reply:
updated

Reply:
Thanks!